spec/models/app_configuration_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.describe AdminOnly::AppConfiguration, type: :model do
+  let(:app_configuration) { create(:app_configuration) }
+
+  describe 'Factory' do
+    it 'has a valid factory' do
+      expect(create(:app_configuration)).to be_valid
+    end
+  end
+
+  describe 'DB Table' do
+    it { is_expected.to have_db_column :id }
+    it { is_expected.to have_db_column :chair_signature_file_name }
+    it { is_expected.to have_db_column :chair_signature_content_type }
+    it { is_expected.to have_db_column :chair_signature_file_size }
+    it { is_expected.to have_db_column :chair_signature_updated_at }
+    it { is_expected.to have_db_column :shf_logo_file_name }
+    it { is_expected.to have_db_column :shf_logo_content_type }
+    it { is_expected.to have_db_column :shf_logo_file_size }
+    it { is_expected.to have_db_column :shf_logo_updated_at }
+  end
+
+  describe 'Validations' do
+    it do
+      is_expected.to validate_attachment_content_type(:chair_signature)
+        .allowing('image/png', 'image/jpeg')
+        .rejecting('image/gif', 'image/bmp')
+    end
+    it do
```

Comment:
need a description for the specification.
Suggestion: "validates attachment content type for SHF logo file"